src/util.rs

```diff
@@ -802,27 +802,31 @@ impl Run {
                 )
             }
             RunMode::Egglog => {
-                assert_eq!(self.eggcc_config.schedule, Schedule::Parallel, "Output egglog only works in parallel mode. Sequential mode does not use a single egraph");
-
                 let rvsdg = Optimizer::program_to_rvsdg(&self.prog_with_args.program)?;
                 let (dag, mut cache) = rvsdg.to_dag_encoding(true);
 
-                let schedule_steps = parallel_schedule();
-                if schedule_steps.len() != 1 {
-                    log::warn!("Parallel schedule had multiple steps! You may need to adjust the schedule to make eggcc tractable.");
-                }
+                let eggcc_config = EggccConfig {
```

Comment:
It would be nice if this took into account stop_after_n_passes so that we can debug particular eggcc passes
Right now it just takes the last pass

Reply:
Yeah realized this right after I pushed